pkg/utils/utils.go

```diff
@@ -269,3 +270,24 @@ func GetArgs(command string, frameWork string, frameworkVersion int,
 
 	return args
 }
+
+// GetTasFilePath returns tas file path
+func GetTasFilePath(path string) (string, error) {
+	path, err := GetConfigFileName(path)
+	if err != nil {
+		return "", err
+	}
+	filePath := fmt.Sprintf("%s/%s", global.RepoDir, path)
+	return filePath, nil
+}
+
+// GenerateUUID generates uuid v4
+func GenerateUUID() string {
```

Comment:
this function may not be required.

Reply:
Need to discuss this , based on earlier comment